rust/src/templates/newvendor.html

```diff
@@ -0,0 +1,115 @@
+                    </div>
+                </div>
+                <div class="form-row justify-content-md-center">
+                    <div class="col-3">
+                    <label for="branche">Branche</label>
+                    </div>
+                    <div class="col">
+                    <input type="text" class="form-control" id="branche" placeholder="Kinderspielzeug">
+                    </div>
+                </div>
+                <div class="form-row justify-content-md-center">
+                    <div class="col-3">
+                    <label for="branche">Kurzbeschreibung des Angebots</label>
+                    </div>
+                    <div class="col">
+                    <textarea class="form-control" id="shortdescription" rows="3"></textarea>
+                    </div>
+                </div>
+                <div class="form-row justify-content-md-center">
+                    <div class="col-3">
+                    <label for="onlineshop">Onlineshop</label>
+                    </div>
+                    <div class="col">
+                        <input type="checkbox" class="form-check-input" id="onlineshop">
+                        <label class="form-check-label" for="onlineshop">Einkauf &uuml;ber Onlineshop m&ouml;glich</label>
+                    </div>
+                </div>
+                <div class="form-row justify-content-md-center">
+                    <div class="col-3">
+                    <label for="website">Webseite</label>
+                    </div>
+                    <div class="col">
+                    <input type="text" class="form-control" id="website" placeholder="http://...">
+                    </div>
+                </div>
+                <div class="form-row justify-content-md-center">
+                    <div class="col-3">
+                    <label for="mail">E-Mail</label>
+                    </div>
+                    <div class="col">
+                    <input type="email" class="form-control" id="email" placeholder="email">
+                    </div>
+                </div>
+                <div class="form-row justify-content-md-center">
+                    <div class="col-3">
+                    <label for="phone">Telefonnummer</label>
+                    </div>
+                    <div class="col">
+                    <input type="numeric" class="form-control" id="phone" placeholder="555-...">
+                    </div>
+                </div>
+                <div class="form-row justify-content-md-center">
+                    <div class="col-3">
+                    <label for="adress">Adresse</label>
+                    </div>
+                    <div class="col-4">
+                    <input type="text" class="form-control" id="address" placeholder="Adresse">
+                    </div>
+                    <div class="col">
+                    <input type="numeric" class="form-control" placeholder="Postleitzahl">
```

Comment:
Type should be `text`, a ZIP code isn't a number in the sense that you can do meaningful arithmetic on it.

Reply:
just wanted to have some simple input verification